lib/destination/dml/columns.go

```diff
@@ -0,0 +1,72 @@
+package dml
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
+	"github.com/artie-labs/transfer/lib/sql"
+	"github.com/artie-labs/transfer/lib/typing"
+	"github.com/artie-labs/transfer/lib/typing/columns"
+)
+
+// UpdateQuery will parse the columns and then returns a list of strings like: cc.first_name=c.first_name,cc.last_name=c.last_name,cc.email=c.email
+func UpdateQuery(c *columns.Columns, dialect sql.Dialect, skipDeleteCol bool) string {
```

Comment:
Everything below this line was copied directly. The only change was to the function definition.